requirements/pre-requirements.txt

```
@@ -1,2 +1,2 @@
-matplotlib~=3.4.3
+matplotlib~=3.5.0rc1
```

This is a pre-release. Why are you using it?
```suggestion
matplotlib~=3.4.3
```

---

At the time of this PR, `matplotlib==3.5.0rc1` was the only release of `matplotlib` that featured support for Python 3.10. Currently, this may not be the case, but that does not matter for two reasons:

1) Any stable release greater than or equal to `3.5.0` will work anyway
2) Merging future Dependabot PRs will fix the issue
3) This is to prepare for an unstable release of epispot anyway